### DISCUSSION TITLE
found a proof, not showing in wallet tho

### DISCUSSION THREAD

**Discussion by tebixi** [Chia-Network/chia-blockchain#6501] [Support]
I found a proof and it shows on my chiaharvestgraph
still the same amount of chia in my wallet though... what is the reason?

**Comment by tebixi** [comment]
![Screenshot from 2021-06-06 10-14-17](https://user-images.githubusercontent.com/84905095/120916385-037e0180-c6b2-11eb-9afe-185efee5b1b4.png)
there is a screenshot for it, the reason is not a slow response time.

**Reply by nonogrub** [reply]
I have the same problem days ago and nobody help me to understand why...

**Comment by BrandtH22** [comment]
@tebixi , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - there is a known bug int he chia client where you need to resync the wallet in order to have the 1.75 portion of claimed pool rewards appear in the client.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
